core/src/main/java/brooklyn/entity/basic/BrooklynConfigKeys.java

```diff
@@ -143,6 +143,7 @@
      * component is up, but this entity does not care about the dependent component's actual config values.
      */
 
+    public static final ConfigKey<Boolean> PROVISION_LATCH = newBooleanConfigKey("provision.latch", "Latch for blocking location provision until ready");
```

Comment:
`SoftwareProcess` redefines the other lifecycle latches (referencing the ones here in `BrooklynConfigKeys`); should `PROVISION_LATCH` also be aliased there for easier discovery? I'm not convinced this is necessarily a good pattern, but inconsistency is worse.